### PR TITLE
Rename 'mask' var to 'keep' in `mask_data_for_fit` and `mask_points_for_fit`

### PR DESCRIPTION
**Pull request by oerc0122**
@abuts 
Changes added to #1424 split out into own PR. 

Unsure of need.

### CODE REVIEW

File: herbert_core/applications/multifit/@mfclass/private/mask_data_for_fit.m [hunk 1]
```diff
@@ -16,9 +16,9 @@
 %   wmask       Cell array (row) of masked datasets.
 %               If there was an error, wmask is an empty cell array size (1,0)
 %
-%   msk_out     Cell array (row) of mask arrays, one per data set.
+%   keep_out    Cell array (row) of mask arrays, one per data set.
 %               Same size as input data.
-%               If there was an error, msk_out is an empty cell array size (1,0)
+%               If there was an error, keep_out is an empty cell array size (1,0)
```
Comment (cmarooney-stfc) [inline herbert_core/applications/multifit/@mfclass/private/mask_data_for_fit.m:21]: It is not at all obvious why keep_out is a better name than mask out. How does keep/msk_out differ from msk_in? Does size here refer to numel(w) only or to sizes of pixel arrays?

File: herbert_core/applications/multifit/@mfclass/private/mask_points_for_fit_xye.m [hunk 2]
```diff
@@ -25,7 +25,7 @@
 %           If give variances, this has the same effect (except that the function cannot check
 %           negative error bars; it does check negative variances of course)
 %
-%   mask    [Optional] array of ones and zeros, with the same number of elements as the data
+%   keep    [Optional] array of ones and zeros, with the same number of elements as the data
```
Comment (cmarooney-stfc) [inline herbert_core/applications/multifit/@mfclass/private/mask_points_for_fit_xye.m:28]: again, why name it keep? what does it do?

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]
Review comments should be met possibly by changing the `keep` names, probably by adding more description to the argument comment lines

File: herbert_core/applications/multifit/@mfclass/private/mask_points_for_fit_xye.m (hunk 2 above)
Comment (abuts) [reply herbert_core/applications/multifit/@mfclass/private/mask_points_for_fit_xye.m:28]: mask presumed removal of marked elements, and here we keep elements which are marked as true. 
This is why renaming occurs. It extreamly difficult to understand code, when you expect one action and it does opposite.

Comment (abuts) [reply herbert_core/applications/multifit/@mfclass/private/mask_points_for_fit_xye.m:28]: at least looking at inconsistency between the method name (`mask`) and the argument name `keep` you think what is actually happening.

Comment (oerc0122) [reply herbert_core/applications/multifit/@mfclass/private/mask_points_for_fit_xye.m:28]: Personally, I disagree, I find the concept of a positive or a negative mask equally valid. However, I think the name `keep` and certainly `keep_out` are worse names than mask. If anything `to_keep` for both would be better, however, I would argue for leaving as-is.

**Pull request closed without merging**